t/010-fifo.t

```diff
@@ -292,6 +292,38 @@ test:test('if_not_exists test', function(test)
     test:isnt(tube, tube_new, "if_not_exists if tube doesn't exists")
 end)
 
+test:test('Space of queue is corrupted', function(test)
+    test:plan(2)
+
+    local fifo = require('queue.abstract.driver.fifo')
+    local space = fifo.create_space('corrupted_fifo_space', { engine = engine })
+    
+    local task_id = space.index.task_id
+    local status = space.index.status
+
+    test:test('task_id index does not exist', function(test)
+        test:plan(2)
+
+        space.index.task_id = nil
```

Comment:
This is not proper use of the index lua api and it is not guaranteed to work in a future. OTOH, I don't see anything that would block usage of \<index_object\>:drop() and then re-creation of the space for a next test.